server/routes/generic/fields.ts

```diff
@@ -499,6 +488,30 @@ const fields = {
       },
     ],
   },
+  whereOutsideHospital: {
```

Comment:
Is this the infamous "or [here a bunch of nested answers]" as an example :)?

May I suggest we only use this conditional fields functionality with the `dateRequired`/`commentRequired` so that we don't accidentally set the wrong expectations? (as we know questions/answers nested within answers opens a whole can of worms)

(but I misunderstood this, disregard this comment ☺️)

Reply:
Yes this is that case. It is simply in here as it was the example I was previously trying to recreate when discussing the generic stuff with Jonathon. 
As we discussed yesterday, we will be looking to only implement text and date inputs as they are required and defined by the NOMIS config. But the functionality is the same, so we will do so because of data management issues etc, but it's there to implement in future if they insist when it comes to new designs.

Reply:
This has been removed now